README.md

```diff
@@ -1,3 +1,7 @@
+# DEPRECATED
+
+This repo is no longer publicly supported.
```

Comment:
Could we specify that this api is no longer supported? We don't use this particular repo.